py/strato/common/log/log2text.py

```diff
@@ -311,15 +370,40 @@ def _getDefaultPaths(self):
         help='show full path to files instead of just module and function')
     parser.add_argument("--withThreads", action="store_true", help='print process and thread name')
     parser.add_argument("-f", "--follow", action="store_true", help='follow file forever', default=False)
-    parser.add_argument("-l", "--localtime", action="store_true", help='print logs in localtime (default utc)', default=False)
+    parser.add_argument("-U", "--utc", action="store_true", help='print logs in utc time (default localtime)', default=False)
     parser.add_argument("--setLocaltimeOffset", type=int, help='set custom localtime offset in hours')
-    parser.add_argument("--restoreLocaltimeOffset", action="store_true", help='restore localtime offset to machine\'s offset')
     parser.add_argument("-i", "--ignoreExtensions", nargs="+",  help="list extensions that you don\'t want to read", default=[".gz"])
     parser.add_argument("-a", "--showAll", action="store_true", help='show all logs', default=False)
-    args = parser.parse_args()
+    parser.add_argument("-n", "--node", type=str, help='run strato-log on remote node (possible input is host name or service with dns resolve', default=None)
+    parser.add_argument("-p", "--password", type=str, help='set default remote password to connect with', default=None)
+    parser.add_argument("-u", "--user", type=str, help='set default remote user to connect to', default=None)
+    parser.add_argument("--restoreLocaltimeOffset", action="store_true", help='restore localtime offset to machine\'s offset')
+    parser.add_argument("-t", "--tail", type=int, help='print n last lines only', default=0)
+
+    args, unknown = parser.parse_known_args()
+    ignoreArgs = []
 
     actionHappened = False
 
+    if args.user != None:
```

Comment:
Please add those 3 (user, password and node) to a separate function: shouldExecuteRemotly (or something similar)